dashboard/client/README.rst

```diff
@@ -6,6 +6,12 @@ Available Scripts
 
 In the project directory, you can run:
 
+``npm ci``
+~~~~~~~~~~~~~~~
+
+Once you've run this command, all the dependencies listed in your package.json
```

Comment:
quote

````
``package.json``
````

?

Reply:
I used the same format that we have for

- [``npm start``](https://github.com/ray-project/ray/pull/39123/files#diff-9bc2a03dfd65ef78f0e0d3154f51ee9fed01634f1e90c0f6f14f5a1611d4528cL9)
- [``npm test``](https://github.com/ray-project/ray/pull/39123/files#diff-9bc2a03dfd65ef78f0e0d3154f51ee9fed01634f1e90c0f6f14f5a1611d4528cR24)
- [``npm run build``](https://github.com/ray-project/ray/pull/39123/files#diff-9bc2a03dfd65ef78f0e0d3154f51ee9fed01634f1e90c0f6f14f5a1611d4528cR32)
- [``npm run eject``](https://github.com/ray-project/ray/pull/39123/files#diff-9bc2a03dfd65ef78f0e0d3154f51ee9fed01634f1e90c0f6f14f5a1611d4528cL39)

We are using `inline literal`, if we change that we should update the rest of the commands

Reply:
on https://github.com/pypa/manylinux

it says `manylinux_x_y` requires `pip >= 20.3`, which is `ALT Linux 10+, RHEL 9+, Debian 11+, Fedora 34+, Mageia 8+, Photon OS 3.0 with updates, Ubuntu 21.04+`